### PR TITLE
Added Cloud Support API Sample Code

### PR DESCRIPTION
**Pull request by eenclona-google**
Added Java code samples for Cloud Support API (v2).

### CODE REVIEW

File: examples/cloud-support/cloud-client/data/updateCase.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (boredabdel) [inline examples/cloud-support/cloud-client/data/updateCase.json:1]: Please add the license header

File: examples/support/cloud-client/data/updateCase.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (boredabdel) [inline examples/support/cloud-client/data/updateCase.json:1]: Please add the license header

File: examples/support/cloud-client/data/case.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (boredabdel) [inline examples/support/cloud-client/data/case.json:1]: Please add the license header

**Review by boredabdel — COMMENTED** [review]
Few comments to solve. Will do an other round of review.

Is there a reason why hidden files (files started with a dot) are included ? Specially the Eclipse ones ?

**Comment by eenclona-google** [comment]
Updated commit to remove hidden files

**Review by boredabdel — APPROVED** [review]
LGTM

**Pull request merged**